### PR TITLE
Add tooltip for subcategory

### LINKED ISSUE #883: [FEATURE] Update UI to make it clear that sub-categories are enabled when Athletics category is selected
**Is your feature request related to a problem? Please describe.**
Currently, it's not clear that subcategories are enabled only for the Athletics category.

**Describe the solution you'd like**
1. Please add the following text as a tooltip to the subcategory label:
`Subcategories are currently available only for Athletics Category.`
2. Please add this to both edit and add events.

### PR DESCRIPTION
**Pull request by clav1d**
## Description
Add tooltip for subcategory to indicate that only Athletics Category have subcategories.

**Fixes #883**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [ ] Within a week
- [x] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

**Comment by bingzhang** [comment]
@clav1d can you check the tooltip text color? and resolve and conflits.

**Comment by clav1d** [comment]
> @clav1d can you check the tooltip text color? and resolve and conflits.

I believe that this issue has been resolved.

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add "Back To Super Event" button for sub events. [#853](https://github.com/rokwire/events-manager/issues/853)
 - Add rich editor for user event description. [#721](https://github.com/rokwire/events-manager/issues/721)
 - Add tooltip on super event and add sub event. [#865](https://github.com/rokwire/events-manager/issues/865)
+- Add tooltip for subcategory to indicate that only Athletics Category have subcategories. [#883](https://github.com/rokwire/events-manager/issues/883)
```
Comment (sandeep-ps) [inline CHANGELOG.md:25]: Please move this to the unreleased section.  Thanks.

**Comment by sandeep-ps** [comment]
Please make the subcategory drop-down enabled when the user selects the Athletics category (in both add and edit HTML files). This is item 1 in linked task #883. Thanks.

**Comment by sandeep-ps** [comment]
> Please make the subcategory drop-down enabled when the user selects the Athletics category (in both add and edit HTML files). This is item 1 in linked task #883. Thanks.

@clav1d, In this PR, I would like to make sure that you have seen my comment above. Thanks.

**Comment by clav1d** [comment]
> > Please make the subcategory drop-down enabled when the user selects the Athletics category (in both add and edit HTML files). This is item 1 in linked task #883. Thanks.
> 
> @clav1d, In this PR, I would like to make sure that you have seen my comment above. Thanks.

Yes, I'm working on this right now.

**Review by sandeep-ps — APPROVED** [review]

**Comment by sandeep-ps** [comment]
Merging this PR.

**Pull request merged**